data/transactions/logic/eval.go

```diff
@@ -344,6 +360,16 @@ func (ep *EvalParams) GetCaller() *EvalContext {
 	return ep.caller
 }
 
+// GetIOBudget returns the current IO budget for the group.
+func (ep *EvalParams) GetIOBudget() uint64 {
+	return ep.ioBudget
+}
+
+// SetIOBudget sets the IO budget for the group.
+func (ep *EvalParams) SetIOBudget(ioBudget uint64) {
+	ep.ioBudget = ioBudget
+}
```

Comment:
Similar situation here: I'm willing to make `ioBudget` public in place of these methods, but I didn't want to start with that because it would pollute the diff.